website/data/users.js

```diff
@@ -0,0 +1,221 @@
+module.exports = [
```

Comment:
I guess it makes more sense to have a `.js` file instead of `.json` file if it is going to be `require`d from code. Is that why you changed it?

Reply:
We can require .json files too. I changed it to JS so that we could add comments into the file and tell people to add new entries alphabetically.